client/components/date-range/README.md

```diff
@@ -0,0 +1,50 @@
+DateRange
+==========
+
+React component used to display a Date Range.
+
+---
+
+## Example Usage
+
+```js
+import React from 'react';
+import DateRange from 'components/date-range';
+
+export default class extends React.Component {
+	<DateRange />
```

Comment:
This example is quite bare and doesn't tell us much, it'd be cool to see some elaboration (perhaps as follow up work) to show some of the more custom usages in the future 👌

Reply:
Also, I've just noticed that this usage isn't valid as `<DateRange />` isn't part of the render method.
Let's either move it into a render method or (preferred) just make this in to a functional component.

Reply:
> This example is quite bare and doesn't tell us much, it'd be cool to see some elaboration (perhaps as follow up work) to show some of the more custom usages in the future

The particular "default" example or the other ones? Was aiming to keep it reasonably simple and encourage others to document use cases as they found them, but happy to update in future as I find them!

> Let's either move it into a render method or (preferred) just make this in to a functional component.

Fixed

Reply:
> Was aiming to keep it reasonably simple and encourage others to document use cases as they found them

That sounds good and fair  👍

Reply:
> The particular "default" example or the other ones?

I just meant the one in the readme, the actual DevDocs examples are great :)